tests/matnormal/test_matnormal_logp_conditional.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+from numpy.testing import assert_allclose
+from scipy.stats import wishart, multivariate_normal
+import tensorflow as tf
+from brainiak.matnormal.utils import rmn
+from brainiak.matnormal.matnormal_likelihoods import (
+                       matnorm_logp_conditional_col,
+                       matnorm_logp_conditional_row)
+from brainiak.matnormal.covs import CovIdentity, CovUnconstrainedCholesky
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+# X is m x n, so A sould be m x p
+
+m = 5
+n = 4
+p = 3
+
+rtol = 1e-7
+
+
+def test_against_scipy_mvn_col_conditional():
```

Comment:
I am not sure if the naming of col_conditional of this function and the test for matnorm_logp_conditional_row is intentional.

Reply:
oh yeah the names are swapped relative to what we're actually testing. fixed.